comment_iowriter.go

```diff
@@ -6,17 +6,50 @@ import (
 	"strings"
 )
 
-var _ CommentService = &CommentWriter{}
+var _ CommentService = &RawCommentWriter{}
 
-type CommentWriter struct {
+// RawCommentWriter is comment writer which writes results to given writer
+// without any formatting.
+type RawCommentWriter struct {
 	w io.Writer
 }
 
-func NewCommentWriter(w io.Writer) *CommentWriter {
-	return &CommentWriter{w: w}
+func NewRawCommentWriter(w io.Writer) *RawCommentWriter {
+	return &RawCommentWriter{w: w}
 }
 
-func (s *CommentWriter) Post(c *Comment) error {
+func (s *RawCommentWriter) Post(c *Comment) error {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported method RawCommentWriter.Post should have comment or be unexported